test/images/redis/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine
+
+RUN apk add --no-cache redis
```

Comment:
It would suck 6 months down the road for there to be a breaking change that breaks tests, would it not? I don't have a particularly strong feeling either way, but if I was forced to choose, I'd say it would be a good idea to lock in a version that works (even if that's current, which looks to be [3.2.5-r0](https://pkgs.alpinelinux.org/packages?name=redis&branch=v3.5&repo=&arch=&maintainer=)